Google.Api.Generator/ProtoUtils/ProtoTyp.cs

```diff
@@ -83,12 +83,16 @@ public static Typ Of(FieldDescriptor desc, bool? forceRepeated = null)
         {
             if (desc.IsMap)
             {
+                var kv = desc.MessageType.Fields.InFieldNumberOrder();
                 if (forceRepeated != null)
                 {
-                    throw new InvalidOperationException("Cannot force repeated on a map field.");
+                    // it turns out that you can force repeated on a map field.
```

Comment:
Nit: please start comment sentences ("it" and "this") with a capital letter. This is a personal bugbear - hope you're okay changing it. (If you feel passionately in the opposite direction, feel free to leave it though...)

Reply:
👍